
#
#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
#spring.datasource.url=jdbc:mysql://sql12.freesqldatabase.com:3306/sql12604584
#spring.datasource.username=sql12604584
#spring.datasource.password=616mpGzyJX
#
spring.jpa.hibernate.ddl-auto=update
spring.jpa.generate-ddl=true
spring.jpa.show-sql=true

server.port=8081

##CLOUD-SQL-CONFIGURATIONS
spring.cloud.gcp.sql.database-name=ecommerce
spring.cloud.gcp.sql.instance-connection-name=singular-backup-380215:asia-southeast1:ecommerce
# So app starts despite "table already exists" errors.
spring.sql.init.continue-on-error=true
# Enforces database initialization
spring.sql.init.mode=always
spring.datasource.username=root
spring.datasource.password=1234
#project id
spring.cloud.gcp.project-id=singular-backup-380215 
spring.cloud.gcp.credentials.location=classpath:com/ptit/springbootdepartmentstore/config/singular-backup-380215-fe6025037758.json


#image store config

gcp.config.file=com/ptit/springbootdepartmentstore/config/singular-backup-380215-fe6025037758.json
gcp.project.id=singular-backup-380215
gcp.bucket.id=image_stored
gcp.dir.name=dev
